src/brand/bhyve/socat

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/perl
```

Comment:
File is duplicating kvm's socat.

Reply:
I'll use the one from kvm then analogous to createzone